components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,276 +15,354 @@ gh_repo = "automate"
+    - `--with-key-sql` is to handle cases where customers have users with multiple pem keys associated with their user or clients. The current chef-server API only dumps the default key. Sometimes, users will generate and assign additional keys to give additional users access to an account but still be able to lock them out later without removing everyone's access.
 
--  Execute the below command to clean unused data from reports. This is an optional steps
+    For example:
+    ```cmd
+        hab pkg exec chef/knife-ec-backup knife ec backup backup_$(date '+%Y%m%d%H%M%s') --webui-key /etc/opscode/webui_priv.pem -s https://chef.io`.
+    ```
 
-``` bash
-    hab pkg exec chef/knife-ec-backup knife tidy server clean --backup-path /path/to/an-ec-backup
-```
+    - Execute the below command to clean unused data from reports. This is an optional step
 
-5.   Execute the below command to copy the backup directory to the Automate HA Chef Server.
-```cmd
-     scp -i /path/to/key backup\_$(date '+%Y%m%d%H%M%s') user@host:/home/user
-```
+    ```bash
+        hab pkg exec chef/knife-ec-backup knife tidy server clean --backup-path /path/to/an-ec-backup
+    ```
 
-## Restore Backed Up Data to Chef Automate HA
+5. Execute the below command to copy the backup directory to the Automate HA Chef Server.
 
--   Execute the below command to install the habitat package for `knife-ec-backup`
+    ```cmd
+        scp -i /path/to/key <backup-file> user@host:/home/user
+    ```
 
-```cmd
-    hab pkg install chef/knife-ec-backup
-```
+    If your HA Chef Server is in a private subnet, scp backup file to bastion and then to Chef Server.
 
--   Execute the below command to restore the backup.
 
-```cmd
-    hab pkg exec chef/knife-ec-backup knife ec restore /home/centos/backup\_2021061013191623331154 -yes --concurrency 1 --webui-key /hab/svc/automate-cs-oc-erchef/data/webui\_priv.pem --purge -c /hab/pkgs/chef/chef-server-ctl/*/*/omnibus-ctl/spec/fixtures/pivotal.rb
-```
+## Adding S3 Configurations for cookbook storage
 
-## In place Migration (Chef Backend to Automate HA)
+Before restoring the backup on the Automate HA Chef Server, If S3 storage is required for cookbooks. Please [Configure S3](/automate/chef_infra_external_cookbooks_in_chef_automate/)
 
-As part of this scenario, customer will migrate from chef-backend (5 machines) to Automate HA in-place, i.e. Automate HA will be deployed in those 5 machines only where Chef-backend is running. One extra bastion node will be required which will be managing the deployment of Automate HA on the chef backend infrastructure.
 
 {{< note >}}
 
-This will require downtime, so plan accordingly. A reduced performance should be expected with this. 
+- If using same bucket for Automate HA , new files will be uploaded for cookbooks. There will not be any impact on the old files.
+- If using the a new bucket fot Auomate HA, new files will be uplaoded for cookbooks
```

Comment:
It has few grammatical error
1. There is a space between comma and HA
2. fot -> for
3 auomate -> automate 
4 uplaoded -> uploaded

The sentence <new files will be uploaded for cookbooks.> is not clear to me